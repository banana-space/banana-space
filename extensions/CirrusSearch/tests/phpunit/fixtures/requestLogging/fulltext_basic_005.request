{
	"type": "fulltext",
	"term": "intitle:main",
	"showSuggestion": false
}
