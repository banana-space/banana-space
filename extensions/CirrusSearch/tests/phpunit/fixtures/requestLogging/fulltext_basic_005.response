[
    {
        "took": 3,
        "timed_out": false,
        "_shards": {
            "total": 4,
            "successful": 4,
            "failed": 0
        },
        "hits": {
            "total": 0,
            "max_score": null,
            "hits": []
        }
    },
    {
        "responses": [
            {
                "took": 72,
                "timed_out": false,
                "_shards": {
                    "total": 8,
                    "successful": 8,
                    "failed": 0
                },
                "hits": {
                    "total": 0,
                    "max_score": null,
                    "hits": []
                }
            }
        ]
    }
]
