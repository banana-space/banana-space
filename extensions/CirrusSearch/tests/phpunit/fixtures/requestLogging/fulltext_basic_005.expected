[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits_offset": 0,
                    "hits_returned": 0,
                    "hits_total": 0,
                    "indices": [
                        "mw_cirrus_metastore"
                    ],
                    "query": "intitle",
                    "query_type": "namespace",
                    "search_time_ms": 3,
                    "suggestion_requested": false
                },
                {
                    "cached": false,
                    "hits_offset": 0,
                    "hits_returned": 0,
                    "hits_total": 0,
                    "indices": [
                        "wiki"
                    ],
                    "limit": 21,
                    "query": "intitle:main",
                    "query_type": "full_text",
                    "search_time_ms": 72,
                    "suggestion_requested": false,
                    "syntax": [
                        "full_text",
                        "full_text_querystring",
                        "complex_query",
                        "intitle"
                    ]
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 0,
            "hitsTotal": 0,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "mw_cirrus_metastore",
            "namespaceName": "intitle",
            "query": "intitle",
            "queryType": "namespace",
            "source": "cli",
            "suggestionRequested": false,
            "tookMs": 0
        },
        "level": "debug",
        "message": "lookup namespace for {namespaceName} against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 0,
            "hitsTotal": 0,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki",
            "limit": 21,
            "query": "intitle:main",
            "queryType": "full_text",
            "source": "cli",
            "suggestionRequested": false,
            "syntax": [
                "full_text",
                "full_text_querystring",
                "complex_query",
                "intitle"
            ],
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]